src/Nethermind/Nethermind.Evm.Precompiles/Snarks/Bn254MulPrecompile.cs

```diff
@@ -18,6 +18,8 @@ public class Bn254MulPrecompile : IPrecompile<Bn254MulPrecompile>
 
     public static Address Address { get; } = Address.FromNumber(7);
 
+    public static string Name => "BN256_MUL";
```

Comment:
Why `BN256`?

Reply:
Simply [due to unknown reason](https://hackmd.io/@liangcc/bls-solidity#alt_bn128-bn256-bn254-why-so-many-names-and-so-confusing). This name is taken from https://eips.ethereum.org/EIPS/eip-7910

Reply:
It will break hash function and therefore the entire purpose of eth_config

Reply:
((we provide hashes of fork configs, so field order, fields names should be fully the same, and no whitesapce))

Reply:
> (unless you change both https://eips.ethereum.org/EIPS/eip-7910 and our code)

I took it seriously: #9023